azurelinuxagent/ga/collect_telemetry_events.py

```diff
@@ -204,8 +204,8 @@ def _capture_extension_events(self, handler_name, handler_event_dir_path):
                     # This is a trade-off between data replication vs data loss.
                     raise
                 except Exception as error:
-                    msg = "Failed to process event file {0}: {1}, {2}".format(event_file, ustr(error),
-                                                                              traceback.format_exc())
+                    msg = "Failed to process event file {0}: \n, {1}".format(event_file,
```

Comment:
I don't think we need these extra \n. The message should display on the same line. as in the previous code.